### ISSUE TITLE
CreepyPasta Generator

### ISSUE THREAD

**Issue by Sleemanmunk** [NaNoGenMo/2018#73]
This is the classic RNN generator approach.
I use the pretrained [ULM-FiT](http://nlp.fast.ai/category/classification.html) model and fine-tune to a corpus of all creepypasta stories. I select the next word randomly weighted by the softmax value of the outputs. I can tweak the likelihood of choosing a less likely word by dividing the raw output by a ratio before I apply softmax. My novel starts at 0.5 (heavy skew towards likely words) and ascends as it progresses to 1.0 to encourage the language to get wilder over time.

**Comment by Sleemanmunk** [comment]
https://github.com/Sleemanmunk/nanogenmo-2018

**Comment by hugovk** [comment]
Oh, I missed labelling this! Congratulations on completing, have a label! I'll re-open it as well to make it more discoverable.

Novel: https://github.com/Sleemanmunk/nanogenmo-2018/blob/master/novel.txt

Preview:

> it in the same , i could n’t get the road , and the street . " i was a man . i was a man . she could feel the side of the tv , and i do n’t know what ’s not really did n’t be , but i was in my face . it ’s not a couple of the dark and the street , and i had been able to his mind , with their face , more . " " i had no one of the fire . " " 
>
> i had a small trunk , i need to meet the old . the morning . " and i was a dream . " " " i was going to the house . i did n’t see a little side of my feet . i was a small ! " he asked , i was the door , and i did n’t even as the house . " i finally believed , and she was n’t know what is it was a few seconds . i was a lot of the room . it ’s when it was a few minutes . " " " i had been a shadow in a bit , and our shoulder . " " " i was a long of a few seconds . " i was no longer . i was n’t believe if the floor . " i was a few minutes . the past , and a bit of the man . he was a lot of the other and the car , he ’s not be the room , and i could n’t have been a couple of the door , " you , i was watching that i know , but i had
